contrib/negdentdelete.py

```diff
@@ -37,7 +37,7 @@ def for_each_child_dentry(dentry: Object) -> Iterator[Object]:
             dentry.d_children.address_of_(),
             "d_sib",
         )
-    except LookupError:
+    except AttributeError:
```

Comment:
What's worse is that I've clearly written this helper before and I got it right everywhere else:
https://github.com/oracle-samples/drgn-tools/blob/main/drgn_tools/dentry.py#L42